tests/db_functions/datetime/test_extract_trunc.py

```diff
@@ -653,7 +653,7 @@ def test_extract_second_func(self):
         self.assertEqual(DTModel.objects.filter(start_datetime__second=ExtractSecond('start_datetime')).count(), 2)
 
     def test_trunc_func(self):
-        start_datetime = datetime(2015, 6, 15, 14, 30, 50, 321)
+        start_datetime = datetime(999, 6, 15, 14, 30, 50, 321)
```

Comment:
This single test seems to test all the paths through `Trunc()` so this change was all that's required to trigger the bug.